hw/ip/alert_handler/doc/_index.md

```diff
@@ -882,7 +882,7 @@ To initialize the block, software running at a high privilege levels (early in t
     - Determine which class (A..D) the alert is associated with.
       Set {{< regref "ALERT_CLASS.CLASS_A" >}} and {{< regref "LOC_ALERT_CLASS.CLASS_LA" >}} accordingly.
 
-    - Optionally lock each alert configuration by writing 0 to {{< regref "ALERT_EN_REGWEN.EN0" >}} or {{< regref "LOC_ALERT_EN_REGWEN.EN0" >}}.
+    - Optionally lock each alert configuration by writing 0 to {{< regref "ALERT_REGWEN.EN0" >}} or {{< regref "LOC_ALERT_REGWEN.EN0" >}}.
```

Comment:
Hey Michael,
Do you think we should use "ALERT_REGWEN0.EN0" instead? Then at least it is linking to an actual register field.
I can apply the change to all related registers if you think it is worth updating it.

Reply:
Ah yes, please go ahead. I did not correctly replace these references...